src/server.c

```diff
@@ -2594,7 +2602,7 @@ void initServer(void) {
         server.db[j].rehashing = listCreate();
         server.db[j].dict_count = slotCount;
         server.db[j].key_count = 0;
-        server.db[j].owned_slots = intsetNew();
+        server.db[j].non_empty_dicts = intsetNew();
```

Comment:
we still only maintain this intset when cluster is disabled, so:
1. let's avoid allocating it (set it to NULL), so if there's any code path that reaches it, it'll segfault and we'll fix it.
2. let's add a comment on that var in the header file.

Reply:
Sounds good.